README.md

```diff
@@ -366,13 +366,9 @@ node.run_state['nginx_configure_flags'] =
   node.run_state['nginx_configure_flags'] | ['--with-SOMETHING', "--with-SOME_OPT='things'"]
 ```
 
-## Maintainers
```

Comment:
would it be valuable for us to start having a boilerplate like this for sous? My first thought is "less code less problems", but this isn't really code...

Reply:
It would be a good thing to add here since it shows up on supermarket